Current build no longer compatible with runelite.

Issue is title.

The problem lies on a recent update of runelite.

```
src\main\java\io\github\deathbeam\plugins\fixedhidechat\FixedHideChatPlugin.java:132: error: cannot find symbol
				client.createScriptEvent(bankWidget.getOnLoadListener())
				      ^
  symbol:   method createScriptEvent(Object[])
  location: variable client of type Client

```

--- a/src/main/java/io/github/deathbeam/plugins/fixedhidechat/FixedHideChatPlugin.java
+++ b/src/main/java/io/github/deathbeam/plugins/fixedhidechat/FixedHideChatPlugin.java
@@ -129,8 +129,9 @@ public class FixedHideChatPlugin extends Plugin implements KeyListener
 			// Causes a very slight flicker of the tag tab above the swap button sadly when opening the bag without the chat hidden
 			if (hideChatPrevious != hideChat)
 			{
-				client.createScriptEvent(bankWidget.getOnLoadListener())
+				client.createScriptEventBuilder(bankWidget.getOnLoadListener())
 					.setSource(bankWidget)
+					.build()
 					.run();
 			}
 			changeWidgetXY(bankWidget, BANK_X);

